README.md

```diff
@@ -240,6 +240,11 @@ Copyright:
  © 2005-2017 Simon Howard.  
 License: [GPL-2.0+](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
 
+Files: `third-party/md5/*`  
+Copyright:  
```

Comment:
Please drop the `Copyright` line for public-domain code.

Reply:
Unrelated, but btw: the win32ports line is missing the Copyright glyph and the trailing dot and two spaces. 😉

Reply:
Thank you!